src/clj/game/core/rezzing.clj

```diff
@@ -26,7 +26,8 @@
       (= :all-costs ignore-cost) [:credit 0]
       alternative-cost alternative-cost
       :else (let [cost (rez-cost state side card {:cost-bonus cost-bonus})
-                  additional-costs (rez-additional-cost-bonus state side card)]
+                  additional-costs (rez-additional-cost-bonus state side card #(or (nil? ignore-cost)
```

Comment:
I don't like this here. Instead, `(when ignore-cost #(= :credit ...))`. Along with the change above, no need to check every step.

Reply:
Ah ya this is much better and makes sense then for the above to use `nil` instead of a true function